forum/settings.py

```diff
@@ -148,6 +149,9 @@
 # Database
 # https://docs.djangoproject.com/en/5.0/ref/settings/#databases
 
+CLIENT = pymongo.MongoClient(os.environ.get('MONGO_URL'), maxPoolSize=400)
+DB = CLIENT[os.environ.get('MONGO_DATABASE')]
+
```

Comment:
We already have Mongo Connection in development. Have you done a merge with develop?
![image](https://github.com/Project-Stage-Academy/UA-1131_Forum-PS/assets/76693557/f2bbabb1-0f55-4fd1-b559-861740d91817)

Reply:
Yes, this connection is gotten from develop. In any case, it's identical to develop branch.